cfn-resources/project/docs/README.md

```diff
@@ -59,6 +99,22 @@ _Type_: <a href="apikeydefinition.md">apiKeyDefinition</a>
 
 _Update requires_: [No interruption](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-updating-stacks-update-behaviors.html#update-no-interrupt)
 
+#### ProjectTeams
```

Comment:
```suggestion
#### ProjectTeam
```

Reply:
We can add multiple Teams to the project, so it is considered to be an array name. ProjectTeams : [] ProjectTeam